docs/_redirects

```diff
@@ -339,3 +339,6 @@ https://www.usehardhat.com/* https://hardhat.org/:splat 301!
 # BDLR902
 /BDLR902 /errors/buidler-errors.html#BDLR902 302
 /bdlr902 /errors/buidler-errors.html#BDLR902 302
+
+# Hardhat Network doc consolidation
+/docs/guides/mainnet-forking.md /docs/hardhat-network/guides/mainnet-forking.md
```

Comment:
Is this working? It doesn't look correct comparing it with the other redirects.

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1657/commits/f85b549f44ab067f215e71adcf804d74e43b8340